source/common/midi_processing.py

```diff
@@ -0,0 +1,174 @@
+import dawdreamer as daw
+from scipy.io import wavfile
+from pydub import AudioSegment
+import os
+
+sample_rate = 44100
+buffer_size = 128
+render_time = 4
+model_midi_path = os.path.join(os.getcwd(), 'files', 'audio_generator', 'midi_files')
+model_preset_path = os.path.join(os.getcwd(), 'files', 'audio_generator', 'preset_files')
+model_wav_path = os.path.join(os.getcwd(), 'files', 'audio_generator', 'wav_files')
+model_plugin_path = os.path.join(os.getcwd(), 'files', 'audio_generator', 'StupidSimpleSampler')      
+
+def drum(list_of_objects, amount_of_drum):
+    for shape in list_of_objects:
+        if amount_of_drum == 0:
+            engine = daw.RenderEngine(sample_rate, buffer_size)
+            synth = engine.make_plugin_processor("my_synth", model_plugin_path + "\\StupidSimpleSampler.dll")
+            assert synth.get_name() == "my_synth"
+            synth.load_state(model_preset_path + "\\kick.fxb")
```

Comment:
same here with the \\

Reply:
@Maucee,
No, This will be addressed in the [Issue: 19-path-fix](https://github.com/Holandsoest/art-to-music/issues/19)